### PR TITLE
Deprecate the STLPort support

### PR DESCRIPTION
**Pull request by var-const**

### CODE REVIEW

File: release_build_files/readme.md [hunk 1]
```diff
@@ -571,13 +571,10 @@ code.
 
 ## Release Notes
 
-### 7.3.0
+### 7.2.0
 -   Changes
     -   General (Android): the STLPort support is deprecated and will be removed
         in the next major release. Please [use libc++ instead](https://developer.android.com/ndk/guides/cpp-support#cs).
```
Comment (jonsimantov) [inline release_build_files/readme.md:577]: "Support for the STLPort STL variant is deprecated..."
Also wrap to 80 (put the link on the next line)?

Comment (jonsimantov) [reply release_build_files/readme.md:577]: Or just copy the warning text exactly, I like that.

**Review by jonsimantov — CHANGES_REQUESTED** [review]

File: release_build_files/readme.md (hunk 1 above)
Comment (var-const) [reply release_build_files/readme.md]: > Or just copy the warning text exactly, I like that.

Done.

Put the whole URL on the next line -- I'm not sure if splitting the markdown element in two could break the formatting?

Comment (jonsimantov) [reply release_build_files/readme.md]: I was thinking wrapping it like this:
 -   General (Android): the STLPort support is deprecated and will be removed
        in the next major release. Please
        [use libc++ instead](https://developer.android.com/ndk/guides/cpp-support#cs).

**Review by jonsimantov — APPROVED** [review]

**Pull request merged**